source/class/qx/ui/table/Table.js

```diff
@@ -1805,8 +1826,9 @@ qx.Class.define("qx.ui.table.Table",
       var col = this.__focusedCol;
       var row = this.__focusedRow;
 
-      // could also be undefined [BUG #4676]
+      // could also be undefined [BUG #4676]. In that case default to first cell focus
       if (col == null || row == null) {
+        this.setFocusedCell(0, 0, true);
```

Comment:
I think using `==` for these two comparisons with null rather than `===` makes this the equivalent of
```
if (! col || ! row)
```
which means that if col is 1 and row is 0, instead, it will use 0, 0 which isn't what's desired...

Reply:
val == null checks if the val is null or undefined since null == undefined. Quick search: https://dorey.github.io/JavaScript-Equality-Table/

Reply:
The specific behavior of `==` for 0, null, and undefined is difficult to remember. My preference here would be to break it out so it's clear what's meant:
```
if (col === null || typeof col == "undefined" || row === null || typeof row == "undefined")
```
That's a bit anal retentive on my part. Maybe just document in a comment above that the comparison for null also compares true for undefined but not for 0?

Reply:
Will change as suggested!